src/integrationTest/java/com/mongodb/hibernate/embeddable/EmptyStructAggregateRetrievalIntegrationTests.java

```diff
@@ -0,0 +1,53 @@
+package com.mongodb.hibernate.embeddable;
+
+
+import com.mongodb.hibernate.junit.MongoExtension;
+import jakarta.persistence.Embeddable;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.hibernate.annotations.Struct;
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.ServiceRegistry;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.hibernate.testing.orm.junit.Setting;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static com.mongodb.hibernate.MongoTestAssertions.assertEq;
+
+@SessionFactory(exportSchema = false)
+@DomainModel(annotatedClasses = {
+        EmptyStructAggregateRetrievalIntegrationTests.Book.class,
+        EmptyStructAggregateRetrievalIntegrationTests.Author.class
+})
+@ExtendWith(MongoExtension.class)
+class EmptyStructAggregateRetrievalIntegrationTests {
+
+    @Test
+    void testEmptyStructAggregateRetriedAsNonNull(SessionFactoryScope scope) {
+        var book = new Book();
+        book.id = 1;
+        book.author = new Author();
+
+        scope.inTransaction(session -> session.persist(book));
+
+        var retrievedBook = scope.fromTransaction(session -> session.get(Book.class, 1));
+        assertEq(book, retrievedBook);
```

Comment:
My understanding is that this test requires `hibernate.create_empty_composites.enabled=true`. If that's true, then it does not ["showcase empty struct could be fetched intact (not null)"](https://github.com/mongodb/mongo-hibernate/pull/105/commits/7032c20b14d0dd12bd1ce24e71676ff676e078fc), as we won't use this configuration property knowing it does not exist in Hibernate ORM 7.0.

We should stop bringing up `hibernate.create_empty_composites.enabled` to avoid confusion.